### ISSUE TITLE
Update message about cache when remote inference disabled

### ISSUE THREAD

**Issue by rbyh** [expectedparrot/edsl#1332]

**Comment by onmyraedar** [comment]
Hi! Is this the reopened version of https://github.com/expectedparrot/coopr/issues/335? Let me know your suggestions and I can help implement them :)

**Comment by rbyh** [comment]
Yes! Just noting to myself that I want to think about what might be useful to explain to a user about caching. I don't have any coherent thoughts yet, so suggestions always appreciated!
